[backport to 3.6] bpo-29474: Improve documentation for weakref.WeakValueDictionary

Mariatta opened the pull request:
There were some grammatical errors in weakref.WeakValueDictionary
documentation.

Same as https://github.com/python/cpython/pull/10, backported to 3.6 branch

eliben commented:
This is a read-only fork of the Python repo. For actual contributions see https://docs.python.org/devguide/index.html

joente commented:
Created issue [Issue25554](http://bugs.python.org/issue25554)

Mariatta committed:
bpo-29474: Improve documentation for weakref.WeakValueDictionary

There were some grammatical errors in weakref.WeakValueDictionary
documentation.

Mariatta commented:
🤔 no travis build here?

zware commented:
The Travis config has not yet been backported.
